### ISSUE TITLE
MSID: 88521 Version: 1 DOI: 2023.04.18.537352

### ISSUE THREAD

**Issue by nlisgo** [elifesciences/enhanced-preprints-import#495]
"msas": "Cell Biology"
"msid": "88521"
"preprintDoi": "10.1101/2023.04.18.537352"

"Reviewed Preprint posted": "2023-07-03"
"Sent for peer review": "2023-04-18"
"Posted to bioRxiv": "2023-06-06" (link: "Go to bioRxiv": "https://www.biorxiv.org/content/10.1101/2023.04.18.537352v3")

[PLACE PDF URL HERE WHEN AVAILABLE] See step 7

## Step 1. Inform bioRxiv

Who can help: @QueenKraken, @nlisgo, @scottaubrey

- [x] Manuscript is in data hub index (https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v1/by-publisher/elife/get-by-manuscript-id?manuscript_id=88521)

## Step 2. Create preview of manuscript

Who can help: @fred-atherden, @nlisgo, @scottaubrey

- [ ] Manuscript is available for preview (https://prod--epp.elifesciences.org/preview/88521/v1)
    - [ ] Trigger import for staging: https://prod--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://prod-elife-epp-data/data
    - [ ] Trigger import for staging: https://staging--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://staging-elife-epp-data/data
    - [ ] Pull request created to [enhanced-preprint-data](https://github.com/elifesciences/enhanced-preprints-data/pulls)
    - [ ] Post link to pull request below

Pull request: https://github.com/elifesciences/enhanced-preprints-data/pull/115

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-manuscript

## Step 3: Awaiting public reviews

Who can help: Editorial team

- [x] Public reviews available on sciety (https://sciety.org/articles/activity/10.1101/2023.04.18.537352)
- [x] Manuscript and editor details at the top of this issue (Supplied by Editorial team)
- [x] Reviews returned by EPP - https://prod--epp.elifesciences.org/api/reviewed-preprints/88521/v1/reviews

<details>
<summary>Example</summary>

```
"msas": "Genetics and Genomics", "Neuroscience"
"msid": "84628"
"version": "1"
"preprintDoi": "10.1101/2022.10.28.514241"
"articleType": "Reviewed Preprint"
"status": "Published from the original preprint after peer review and assessment by eLife."

"Reviewed Preprint posted": "2023-01-02"
"Sent for peer review": "2022-10-28"
"Posted to bioRxiv": "2022-11-21" (link: "Go to bioRxiv": "https://www.biorxiv.org/content/10.1101/2022.10.28.514241v1")

Editors:
Reviewing Editor
Michael B Eisen
University of California, Berkeley, United States
Senior Editor
Michael B Eisen
University of California, Berkeley, United States
```

</details>

## Step 4: Modify manuscripts.json (no PDF)

- [ ] Reviewed preprint is published to EPP (https://elifesciences.org/reviewed-preprints/88521)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below
- [ ] Request that a doi be registered
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

Pull request: [PLACE LINK TO PULL REQUEST HERE]
[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

<details>
<summary>Instructions to modify manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/publish-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/334/files

Once the pull request is merged in it should be available a few minutes later.

</details>

### Request that a doi

Post the following in [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77):

```
@Fred can you register a doi for https://elifesciences.org/reviewed-preprints/88521
```

## Step 5: Awaiting search reindex

- [ ] Reviewed preprint is avaliable on journal homepage (https://elifesciences.org)

The search reindex is triggered once an hour. We need the reviewed preprint to be indexed as the search application serves the journal homepage.

<details>
<summary>Additional info</summary>

If needed, the jenkins pipeline to reindex search can be triggered sooner.

https://alfred.elifesciences.org/job/process/job/process-reindex-reviewed-preprints/

</details>

## Step 6: Published! Request PDF generation

- [ ] Post the link to the PDF url at the top of the issue
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

Post the following to the [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) on slack:

```
@Ryan Dix-Peek please can you generate a PDF for https://elifesciences.org/reviewed-preprints/88521
```

## Step 7: Introduce PDF to data folder and git repo

- [ ] PDF is avaliable at https://github.com/elifesciences/enhanced-preprints-data/raw/master/data/88521/v1/88521-v1.pdf

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-pdf

## Step 8: Add PDF url to manuscripts.json

- [ ] Reviewed preprint PDF is available for download (https://prod--epp.elifesciences.org/reviewed-preprints/88521)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below

[PLACE LINK TO PULL REQUEST HERE]

<details>
<summary>Instructions to add PDF url to manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/add-pdf-url-to-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/397/files

Once the pull request is merged in it should be available a few minutes later.

</details>

## Step 9: Done!

- [ ] Kettle is on!

**Comment by fred-atherden** [comment]
We have version 2 loaded instead of version 3.

**Comment by fred-atherden** [comment]
@acollings, it looks like the authors have included their cover letter in the preprint. Should we alter the authors and give them an an opportunity to publish a new version of the preprint (that we will then pull in)? Or should we proceed as is?



> Dear Editor,
> 
> We would like to submit our manuscript titled “BATF relieves hepatic steatosis by inhibiting PD1 and promoting energy metabolism”, to be considered for publication in elife.
> 
> Non-alcoholic fatty liver disease (NAFLD) has become the most common chronic liver disease in the world. Its estimated global prevalence is about 25%, and that number is quickly growing. Therefore, it is urgent to understand the pathogenesis of NAFLD, which will be essential for its prevention and treatment.
> 
> BATF is generally considered to be associated with immune disorders, but the effect of BATF on lipid metabolism has not been clear. In this paper, our data indicated that BATF alleviates high-fat diet-induced hepatic steatosis. And BATF can inhibit the elevated PD1 expression induced by high-fat diet. Mechanism study confirms that BATF regulates fat accumulation by inhibiting PD1 and promoting energy metabolism. Moreover, the use of PD1 antibodies alleviates hepatic lipid deposition. In conclusion, we identified the regulatory role of BATF on hepatic lipid metabolism and found the target PD1 for the alleviation of NAFLD. This provides new insights into NAFLD and immunotherapy hotspot PD1, and closes the relationship between immunity and metabolism.
> 
> Currently, there is no FDA-approved drug for NAFLD. However, the therapeutic approach of many cancer focuses on PD1, and the related drugs have been developed. Hence, our results will expand the application of PD1-targeted drugs in the treatment of NAFLD and provide a new strategy to prevent hepatic steatosis and decrease body weight.
> 
> We state that no data reported in this manuscript has been published elsewhere or is under consideration for publication elsewhere. All authors have seen the manuscript and approved to submit to your journal. We state there is no conflict of interest.
> 
> Our e-mail address is: `zhoulei@gxu.edu.cn`
> 
> Our address is: Guangxi Academy of Medical Sciences, the People’s Hospital of Guangxi Zhuang Autonomous Region, Nanning, 530021, China
> 
> Thank you very much for your attention and consideration.
> 
> Sincerely yours,
> 
> Lei Zhou
> 
> The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.

**Comment by acollings** [comment]
Thanks @fred-atherden I'll ask the authors what they'd prefer.

**Comment by acollings** [comment]
Actually, @fred-atherden the authors have resubmitted, so they'll be prompted to post a revised preprint, and I wonder if this request will confuse things at this stage. As the letter is at the end, I'm comfortable proceeding, if you are. (Though it would be good to avoid this in the revised version!)

**Comment by fred-atherden** [comment]
Thanks @acollings. I thin I'll remove it from the version we publish (Fix is in https://github.com/elifesciences/enhanced-preprints-biorxiv-xslt/pull/54).
